storage/object/how-to/get-started-edge-services.mdx

```diff
@@ -0,0 +1,293 @@
+    <Lightbox src="scaleway-bucket-name-in-the-list.webp" alt="" />
+
+3. Click the **Edge Services** tab.
+
+    TODO SCREENSHOT
+
+4. Click **Enable Edge Services for free**.
+
+    A pop-up informs you that your bucket will be exposed via Edge Services and that you can disable Edge Services at any time. Edge Services is free during the beta stage, and you will be notified before it becomes billable.
+
+5. Click **Enable Edge Services** to confirm.
+
+    Edge Services is enabled on your bucket, and its various features are now visible on the Edge Services tab, ready to be configured.
+
+    TODO SCREENSHOT
+
+    <Message type="important">
+
+    Your bucket's [visibility](https://www.scaleway.com/en/docs/storage/object/concepts/#visibility) can be set to **private**, but any objects within it that you want to expose via Edge Services must be set to [**public** visibility](https://www.scaleway.com/en/docs/storage/object/how-to/manage-object-visibility/). However, in the case that you are using Edge Services with bucket website (see below), objects can remain private.
+
+    </Message>
+
+    <Message type="important">
+    Note the following important information about using Edge Services with the [bucket website](/storage/object/how-to/use-bucket-website/) feature:
+    - **If you want to use bucket website and Edge Services**: Ensure that you activate bucket website **before** you enable Edge Services. Avoid deactivating bucket website afterwards.<br /><br />
+    - **If you do not want to use bucket website**: Simply enable Edge Services. Note however that if you activate bucket website after this, Edge Services will not be able to serve your bucket website URLs, and will continue to serve the regular bucket URLs.<br /><br />
+    - **If you want to change your bucket website setting**: Disable Edge Services first, activate or deactivate bucket website as you wish, and then enable Edge Services. Edge Services will now pick up the new configuration, and will serve content from your bucket website URLS if bucket website is activated, or from your regular bucket URLs if bucket website is deactivated.  
+    </Message>
+
+## How to access your bucket via Edge Services
+
+Once you have enabled Edge Services on your bucket, you can access your bucket and its content via the following endpoints
+
+| Endpoint                                | Where to find this endpoint in the console | Notes                                                                |
+|-----------------------------------------|--------------------------------------------|----------------------------------------------------------------------|
+| `https://bucket-name.s3.nl-ams.scw.cloud` | The **Bucket settings** tab                | Edge Services is bypassed when bucket is accessed via this endpoint  |
+| `https://bucket-name.edge.scw.cloud`      | The **Edge Services** tab                  | Edge Services serves bucket content when this endpoint is used |
```

Comment:
@RoRoJ the endpoint is actually different, it's https://bucket-name.svc.edge.scw.cloud
Sorry about the change